MvvmCross/iOS/iOS/Views/Presenters/MvxIosViewPresenter.cs

```diff
@@ -424,7 +445,7 @@ protected virtual void SetWindowRootViewController(UIViewController controller)
             foreach (var v in _window.Subviews)
                 v.RemoveFromSuperview();
 
-            _window.AddSubview(controller.View);
+            //_window.AddSubview(controller.View);
             _window.RootViewController = controller;
```

Comment:
I think it is good practise not to leave commented out code in the code base. Just remove it, or if there is a good reason to leave it mention it with a comment in the code so other people also know why it is commented out.